ignite/templates/app/stargate/config.yml

```diff
@@ -12,6 +12,10 @@ client:
     path: "docs/static/openapi.yml"
   typescript:
     path: "ts-client"
+  composables:
+    path: "vue/src/composables"
+  hooks:
+    path: "react/src/hooks"
```

Comment:
Users should opt-in on generated code features so we should remove these from the template. Values can be added by users if they choose to opt-in or when they generate the code using generation commands.

```suggestion
```

Reply:
Agreed. Just stuck with the existing pattern so as not to solve all the issues discussed here: https://github.com/ignite/cli/pull/2997 in this PR :P